services/brig/src/Brig/Data/User.hs

```diff
@@ -313,6 +313,9 @@ lookupAuth u = fmap f <$> retry x1 (query1 authSelect (params Quorum (Identity u
   where
     f (pw, st) = (pw, fromMaybe Active st)
 
+-- | Return users with given IDs.
+--
+-- Skips nonexistent users. /Does not/ skip users who have been deleted.
```

Comment:
I would argue that's surprising enough to justify giving this function the name `lookupUsersIncludeDeleted`.  This would also force us to go around and make sure all callers seem to know what they're doing.